apps/full-site-editing/full-site-editing-plugin/starter-page-templates/class-starter-page-templates.php

```diff
@@ -147,4 +114,26 @@ public function enqueue_styles() {
 			filemtime( plugin_dir_path( __FILE__ ) . 'dist/' . $style_file )
 		);
 	}
+
+	/**
+	 * Fetch vertical data from the API or return cached version if available.
+	 */
+	public function fetch_vertical_data() {
+		$vertical_templates = get_transient( self::$transient_key );
```

Comment:
I think would be fine to keep the transient key within this method. It shouldn't be of interest outside of it.